### DISCUSSION TITLE
2023-09-18

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#525] [daily_meeting]
### 오늘의 할 일
프론트엔드
- 레이아웃 컴포넌트 미션
- 
백엔드
- 5차 필수 요구사항 만족하기

### 개인 일정
노아 룩소 마코 : 3-4시 근로

### 내일의 할 일

**Comment by jaehee329** [comment]
### 체크인
8점 - 선릉 마지막주... 잘 해보죠
### 체크아웃
7점 - nGrinder에 갈려버린 우리..

**Comment by aak2075** [comment]
### 체크인

10점 - 버스타고 쾌적하게 와서 기분이 좋네요

### 체크아웃

10점 - 삽질만 열심히 했네요

**Comment by yeopto** [comment]
### 체크인

10점 - 활기차게 한 주를 시작해봐요

### 체크아웃

10점 - 하루가 정말 빨리 갔네요

**Comment by woosung1223** [comment]
### 체크인
8점 - 무난한 월요일입니다

### 체크인
8점 - 허브가 Locust를 도와줘서 좋네요

**Comment by nlom0218** [comment]
## 체크인

10점 - 주말에 아주 그냥 푹 쉬었더니 좋네요 좋아요 👍

## 체크아웃

7점 - 미션 제출을 못하는 줄 알고 무서웠어요

**Comment by woo-jk** [comment]
### 체크인
8점

### 체크아웃
9점 - 문제없이 미션 끝내서 좋아요~!

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 주말에 잘 쉬었는데 발표 생각하니 가슴이 미어지네요...

### 체크아웃
6점 - ngrinder에 갈려버렸읍니다...
